src/components/common/DeprecatedUpdateWarn.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import triangleAlert from '../../assets/icons/ic-alert-triangle.svg';
+
+export class DeprecatedWarn extends Component {
+    render() {
+        return (
+            <div className="flexbox" style={{alignItems: 'center'}}>
+                <img src={triangleAlert} className="icon-dim-16 mr-8" style={{margin: 0}}/>
```

Comment:
you are setting margin: 0, then why using mr-8 ?

Reply:
removed mr-8